src/include/utils/resgroup.h

```diff
@@ -235,6 +235,9 @@ extern Oid ResGroupGetGroupIdBySessionId(int sessionId);
 extern char *getCpuSetByRole(const char *cpuset);
 extern void checkCpuSetByRole(const char *cpuset);
 
+extern void selfAddStartupChunks(int32 chunks);
+extern void selfSubStartupChunks(int32 chunks);
```

Comment:
I think we should rename public functions to something like ResGroupAddStartupChunks and ResGroupSubStartupChunks